.ghc.environment.x86_64-darwin-8.6.4

```diff
@@ -0,0 +1,86 @@
+-- This is a GHC environment file written by cabal. This means you can
```

Comment:
wtf. I did not have this file locally.

Reply:
🔥

Reply:
`git rm --cached .ghc.environment.x86_64-darwin-8.6.4` should leave the file locally but remove it from the repo. I added it to the `.gitignore` file on `master` too, so it shouldn’t try to re-add it.

Reply:
Never mind, the files changed tab was showing me stale data!

Reply:
Np!